mandatory/1-fix-functions.js

```diff
@@ -31,8 +31,8 @@ function greaterThan10(num) {
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+  let index = [3];
```

Comment:
`let index = 3` index is a type `number` it shouldn't be an array as above. Please review your solution.
hint: `arr[index]` will return the element at the `index` position of the array `arr`